.storybook/main.js

```diff
@@ -9,11 +9,7 @@ module.exports = {
     "../frontend/**/*.stories.mdx",
     "../frontend/**/*.stories.@(js|jsx|ts|tsx)",
   ],
-  addons: [
-    "@storybook/addon-essentials",
-    "@storybook/addon-links",
-    "@storybook/addon-a11y",
-  ],
+  addons: ["@storybook/addon-essentials", "@storybook/addon-links"],
```

Comment:
a11y addon freezes storybook when MultiSelect is added, there is no workaround I could find

Reply:
Wild. Any theory as to why? Would like to be able to re-add it eventually.

Reply:
I've updated the underlying library and it seems to work now, so this change is reverted